docs/source/admin_guide/backup.md

```diff
@@ -63,6 +63,8 @@ spec:
           name: volume-to-debug-ubuntu
 ```
 
+> Note in QHub versions before v0.4 replace `claimName: "jupyterhub-dev-share"` with `claimName: "nfs-mount-dev-share"`
+
 Run:
 
 ```shell
```

Comment:
Suggestion replace `dev` with <your-namespace> from here on

Reply:
Then you can remove some bits referring to the change from `dev` to other namespaces

Reply:
`dev` isn't referring to the namespace in this case, but I see the confusion so will explain.